rpc/internal/test_results_service.go

```diff
@@ -74,13 +82,15 @@ func (s *testResultsService) AddTestResults(ctx context.Context, results *TestRe
 		return nil, newRPCError(codes.Internal, errors.Wrapf(err, "problem appending test results for '%s'", results.TestResultsRecordId))
 	}
 
-	// for _, res := range exportedResults {
-	//     grip.Error(message.WrapError(s.env.GetRemoteQueue().Put(ctx, units.NewHistoricalTestDataJob(s.env, record.Info, res)), message.Fields{
-	//         "message":     "failed to enqueue historical test data job",
-	//         "info":        record.Info,
-	//         "test_result": res,
-	//     }))
-	// }
+	if record.Info.HistoricalTestData {
+		for _, res := range exportedResults {
+			grip.Error(message.WrapError(s.env.GetRemoteQueue().Put(ctx, units.NewHistoricalTestDataJob(s.env, record.Info, res)), message.Fields{
```

Comment:
Do we still need this job? I thought it was going to be deleted or just in-lined. It would be much more efficient if it was inlined given that it has to run really often.

Reply:
I was going to get rid of the job entirely in the next  ticket, EVG-14440. Unless you think we should keep the job and just run it inline?

Reply:
I also would prefer to get rid of the job, but I don't think we can uncomment this block until we remove the job, or else the deploy will include this code.